authserver/oauth2/rfc6749/authorization_server.py

```diff
@@ -72,7 +108,13 @@ def create_token_response(self, request=None):
 
         try:
             grant.validate_token_request()
-            args = grant.create_token_response()
-            return self.handle_response(*args)
+            status, body, headers = grant.create_token_response()
+
+            bh_jwt = generate_jwt(body['access_token'])
```

Comment:
@gregmundy this _seems_ to be okay based on all the code I went through in authlib.